examples/test/github_issue_mapping.json

```diff
@@ -0,0 +1 @@
ub.com/users/sbose78/gists{/gist_id}","starred_url":"https://api.github.com/users/sbose78/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/sbose78/subscriptions","organizations_url":"https://api.github.com/users/sbose78/orgs","repos_url":"https://api.github.com/users/sbose78/repos","events_url":"https://api.github.com/users/sbose78/events{/privacy}","received_events_url":"https://api.github.com/users/sbose78/received_events","type":"User","site_admin":false},"assignees":[{"login":"sbose78","id":545280,"avatar_url":"https://avatars.githubusercontent.com/u/545280?v=3","gravatar_id":"","url":"https://api.github.com/users/sbose78","html_url":"https://github.com/sbose78","followers_url":"https://api.github.com/users/sbose78/followers","following_url":"https://api.github.com/users/sbose78/following{/other_user}","gists_url":"https://api.github.com/users/sbose78/gists{/gist_id}","starred_url":"https://api.github.com/users/sbose78/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/sbose78/subscriptions","organizations_url":"https://api.github.com/users/sbose78/orgs","repos_url":"https://api.github.com/users/sbose78/repos","events_url":"https://api.github.com/users/sbose78/events{/privacy}","received_events_url":"https://api.github.com/users/sbose78/received_events","type":"User","site_admin":false}],"milestone":{"url":"https://api.github.com/repos/almighty/almighty-core/milestones/3","html_url":"https://github.com/almighty/almighty-core/milestones/Sprint%20%23119","labels_url":"https://api.github.com/repos/almighty/almighty-core/milestones/3/labels","id":1934545,"number":3,"title":"Sprint #119","description":"","creator":{"login":"aslakknutsen","id":132158,"avatar_url":"https://avatars.githubusercontent.com/u/132158?v=3","gravatar_id":"","url":"https://api.github.com/users/aslakknutsen","html_url":"https://github.com/aslakknutsen","followers_url":"https://api.github.com/users/aslakknutsen/followers","following_url":"https://api.github.com/users/aslakknutsen/following{/other_user}","gists_url":"https://api.github.com/users/aslakknutsen/gists{/gist_id}","starred_url":"https://api.github.com/users/aslakknutsen/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/aslakknutsen/subscriptions","organizations_url":"https://api.github.com/users/aslakknutsen/orgs","repos_url":"https://api.github.com/users/aslakknutsen/repos","events_url":"https://api.github.com/users/aslakknutsen/events{/privacy}","received_events_url":"https://api.github.com/users/aslakknutsen/received_events","type":"User","site_admin":false},"open_issues":18,"closed_issues":6,"state":"open","created_at":"2016-08-10T11:48:26Z","updated_at":"2016-08-24T04:40:40Z","due_on":"2016-09-06T00:00:00Z","closed_at":null},"comments":19,"created_at":"2016-08-17T10:26:46Z","updated_at":"2016-08-23T13:45:45Z","closed_at":null,"body":"related https://trello.com/c/YNeXoM2R/103-create-remoteissue-to-workitemtype-mapping-model","closed_by":null}
```

Comment:
The `test` directory would kind of differentiate the file from other examples in a way that it would imply that the code consumes the file and hence should not be touched without reason :)